internal/apps/poll/restrict.go

```diff
@@ -36,6 +36,29 @@ func RestrictPoll(r restricter.HasPermer, canSee, canManage string, restrictedFi
 		}
 		poll["user_has_voted"] = hasVoted
 
+		// get the users `vote_delegated_from_users_id`
+		var user struct {
+			VoteDelegationIds []int `json:"vote_delegated_from_users_id"`
+		}
+		if err := r.Get("users/user", uid, &user); err != nil {
+			return nil, fmt.Errorf("unmarshal user: %w", err)
+		}
+		// Calc the intersection of voteDelegationIds and votedID
+		hasVotedForDelegationIds := []int{}
```

Comment:
Please use a shorter variable name. If the scope of the variable is only some lines the go style (that I use in this repo) says, that a short variable name is better.

https://github.com/golang/go/wiki/CodeReviewComments#variable-names


Please initialize the variable with:

```
var ids []int
```